scrnaseq.Rmd

```diff
@@ -67,7 +67,8 @@ knitr::opts_chunk$set(echo=TRUE,                     # output code
                       dev=c('png', 'pdf'),           # create figures in png and pdf; the first device (png) will be used for HTML output
                       dev.args=list(png=list(type="cairo"),  # png: use cairo - works on cluster, supports anti-aliasing (more smooth)
                                     pdf=list(bg="white")),     # pdf: use cairo - works on cluster, supports anti-aliasing (more smooth)
-                      dpi=96                         # figure resolution
+                      dpi=96,                         # figure resolution
+                      fig.retina=2                   # DISCUSS: http://zevross.com/blog/2017/06/19/tips-and-tricks-for-working-with-images-and-figures-in-r-markdown-documents/#the-fig.retina-argument-is-a-resolution-multiplier
```

Comment:
Here is the same report compiled with fig.retina=1 (96dpi) and fig.retina=2 (182dpi): https://cloudstore.zih.tu-dresden.de/index.php/s/2FzxDijSQsnZ8r9

The report with fig.retina=2 is twice as large. But, if you zoom in on fig.retina=1 pictures you will see some differences. I would say that I do not need high resolution since there is always a pdf available (which is independent of resolution).

Reply:
Mmmh, what is the actual size of the HTMLs? I prefer `fig.retina=2` over `fig.retina=1`.